assets/src/edit-story/components/canvas/gridview/index.js

```diff
@@ -73,21 +91,56 @@ const Rectangle = styled.div`
   }
 `;
 
-function RangeControl({ value, onChange }) {
+// I think we can move this component to shared one in the future if we need this kind of spacer span.
+const SpacerSpan = styled.span`
```

Comment:
In other places we just use something like this:

```js
const Space = styled.div`
  width: 20px;
`;
```

Reply:
Okay

Reply:
Or even just `flex: 0 0 20px`, which will make it work in a flex container in any direction.

Reply:
Fixed